schema/site_metadata.json

```diff
@@ -0,0 +1,74 @@
+{
+    "$udmi_version": "1.5.0",
+    "title": "Site Metadata",
+    "$section": "Site Model",
+    "type": "object",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "additionalProperties": false,
+    "required": [
+      "timestamp",
+      "version"
+    ],
+    "properties": {
+      "timestamp": {
+        "description": "RFC 3339 timestamp the site metadata was last updated",
+        "type": "string",
+        "format": "date-time",
+        "examples": ["2019-01-17T14:02:29.364Z"]
+      },
+      "version": {
+        "description": "Version of the UDMI schema",
+        "type": "string"
+      },
+      "id": {
+        "description": "Identifier for the site or building",
+        "type": "string",
+        "examples": ["UK-LON-ABC", "US-NYC-HUD1"]
+      },
+      "name": {
+        "description": "Name of the site or building",
+        "type": "string",
+        "examples": ["Cheesegrater", "Mountain View"]
+      },
+      "location": {
+        "type": "object",
+        "properties": {
+            "address": {
+                "description": "Postal address of the site",
+                "type": "string",
+                "examples": ["1600 Amphitheatre Parkway, Mountain View, CA 94043, United States"]
+            },
+            "lat": {
+                "description": "Latitude of the site in WGS84 coordinates, as indicated by a map marker ",
+                "type": "number"
+            },
+            "lon": {
+                "description": "Longitude of the site in WGS84 coordinates, as indicated by a map marker",
+                "type": "number"
+            }
+          }
+      },
+      "origin": {
```

Comment:
ho is "origin" different than "location"?

Reply:
Location would be the marker on Google Maps, and would usually be the entrance or centre of the building. Origin would be the site origin datum reference used in the BIM modelling/CAD, against which all x,y,z references are against, which will usually be a corner of the site.

Reply:
for my own sanity, maybe add a brief "description" each that explains this :-)